community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/ast/rewriters/literalReplacement.scala

```diff
@@ -35,9 +36,16 @@ object literalReplacement {
   }
 
   private val literalMatcher: PartialFunction[Any, LiteralReplacements => (LiteralReplacements, Option[LiteralReplacements => LiteralReplacements])] = {
-    case _: ast.Match | _: ast.Create | _: ast.CreateUnique | _: ast.Merge | _: ast.SetClause | _: ast.Return | _: ast.With =>
+    case _: ast.Match |
+         _: ast.Create |
+         _: ast.CreateUnique |
+         _: ast.Merge |
+         _: ast.SetClause |
+         _: ast.Return |
+         _: ast.With |
+         _: ast.CallClause =>
       acc => (acc, Some(identity))
-    case _: ast.Clause | _: ast.PeriodicCommitHint | _: ast.Limit | _: ast.CallProcedure =>
+    case _: ast.Clause | _: ast.PeriodicCommitHint | _: ast.Limit =>
```

Comment:
Either don't line up them like you did above, or line up this `case` as well. The disharmony between these two blocks is itching